lib/zendesk_apps_support/validations/requirements.rb

```diff
@@ -92,23 +107,22 @@ def invalid_channel_integrations(requirements)
           end
         end
 
-        def invalid_custom_resources_schema(requirements)
-          custom_resources_schema = requirements['custom_resources_schema']
-          return unless custom_resources_schema
-          valid_schema_keys = %w[resource_types relationship_types]
+        def invalid_custom_objects(requirements)
+          custom_objects = requirements['custom_objects']
+          return unless custom_objects
```

Comment:
TODO: error if `requirements['custom_objects']` is `false` rather than just returning